--- a/tests/cefsimple/cefsimple_mac.mm
+++ b/tests/cefsimple/cefsimple_mac.mm
@@ -164,6 +164,13 @@
 
     // Specify CEF global settings here.
     CefSettings settings;
+    const std::string profile_path =
+        command_line->GetSwitchValue("user-data-dir");
+    if (!profile_path.empty()) {
+      CefString(&settings.root_cache_path) = profile_path;
+      CefString(&settings.cache_path) = profile_path;
+      settings.persist_session_cookies = true;
+    }
 
     // When generating projects with CMake the CEF_USE_SANDBOX value will be
     // defined automatically. Pass -DUSE_SANDBOX=OFF to the CMake command-line
