--- a/tests/cefsimple/mac/English.lproj/MainMenu.xib
+++ b/tests/cefsimple/mac/English.lproj/MainMenu.xib
@@ -10,10 +10,10 @@
         <customObject id="-3" userLabel="Application"/>
         <menu title="AMainMenu" systemMenu="main" id="29" userLabel="MainMenu">
             <items>
-                <menuItem title="cefsimple" id="56">
+                <menuItem title="Saccade" id="56">
                     <menu key="submenu" title="TestShell" systemMenu="apple" id="57">
                         <items>
-                            <menuItem title="About cefsimple" id="58">
+                            <menuItem title="About Saccade" id="58">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <connections>
                                     <action selector="orderFrontStandardAboutPanel:" target="-2" id="142"/>
@@ -32,7 +32,7 @@
                             <menuItem isSeparatorItem="YES" id="144">
                                 <modifierMask key="keyEquivalentModifierMask" command="YES"/>
                             </menuItem>
-                            <menuItem title="Hide cefsimple" keyEquivalent="h" id="134">
+                            <menuItem title="Hide Saccade" keyEquivalent="h" id="134">
                                 <connections>
                                     <action selector="hide:" target="-1" id="367"/>
                                 </connections>
@@ -51,7 +51,7 @@
                             <menuItem isSeparatorItem="YES" id="149">
                                 <modifierMask key="keyEquivalentModifierMask" command="YES"/>
                             </menuItem>
-                            <menuItem title="Quit cefsimple" keyEquivalent="q" id="136" userLabel="1111">
+                            <menuItem title="Quit Saccade" keyEquivalent="q" id="136" userLabel="1111">
                                 <connections>
                                     <action selector="terminate:" target="-1" id="369"/>
                                 </connections>
@@ -315,7 +315,7 @@
                 <menuItem title="Help" id="103" userLabel="1">
                     <menu key="submenu" title="Help" id="106" userLabel="2">
                         <items>
-                            <menuItem title="cefsimple Help" keyEquivalent="?" id="111">
+                            <menuItem title="Saccade Help" keyEquivalent="?" id="111">
                                 <connections>
                                     <action selector="showHelp:" target="-1" id="360"/>
                                 </connections>
