--- a/tests/cefsimple/mac/English.lproj/MainMenu.xib
+++ b/tests/cefsimple/mac/English.lproj/MainMenu.xib
@@ -23,6 +23,6 @@
                             <menuItem isSeparatorItem="YES" id="236">
                                 <modifierMask key="keyEquivalentModifierMask" command="YES"/>
                             </menuItem>
-                            <menuItem title="Preferences…" keyEquivalent="," id="129" userLabel="121"/>
+                            <menuItem title="Settings…" keyEquivalent="," id="129" userLabel="121"/>
                             <menuItem isSeparatorItem="YES" id="143">
                                 <modifierMask key="keyEquivalentModifierMask" command="YES"/>
