--- a/tests/cefsimple/simple_handler.cc
+++ b/tests/cefsimple/simple_handler.cc
@@ -157,7 +157,7 @@ bool SimpleHandler::OnBeforePopup(
   CEF_REQUIRE_UI_THREAD();
   if (IsSaccadeTabDisposition(target_disposition) &&
       !target_url.ToString().empty()) {
-    SaccadeAdapter::GetInstance()->OpenUserTabOnUi(target_url.ToString());
+    SaccadeAdapter::GetInstance()->OpenRoutedTabOnUi(target_url.ToString());
     return true;
   }
   return false;
@@ -173,7 +173,7 @@ bool SimpleHandler::OnOpenURLFromTab(
   CEF_REQUIRE_UI_THREAD();
   if (IsSaccadeTabDisposition(target_disposition) &&
       !target_url.ToString().empty()) {
-    SaccadeAdapter::GetInstance()->OpenUserTabOnUi(target_url.ToString());
+    SaccadeAdapter::GetInstance()->OpenRoutedTabOnUi(target_url.ToString());
     return true;
   }
   return false;
